client/overview.jsx

```diff
@@ -29,7 +29,7 @@ class Overview extends React.Component {
   }
 
   fetchOverview() {
-    return axios.get('/overview/all', { params: { campId: this.state.campId } })
+    return axios.get('http://localhost:3003/overview/all', { params: { campId: this.state.campId } })
```

Comment:
Oh neat, is this your request to your proxy?

Reply:
no it's just a request directly to my service. I'm not sure if it's the right thing to do, but once I got my proxy running I saw that when i made the simple `/overview/all` request it was trying to make it on the proxy's url and I wanted to make it of my service url.